src/Model/Event.php

```diff
@@ -11,6 +11,8 @@
 
 namespace CalendArt\Adapter\Office365\Model;
 
+use CalendArt\Adapter\Office365\TimezoneInterface;
+use CalendArt\Adapter\Office365\WindowsTimezone;
```

Comment:
Please use the same notation (I'll convert everything in one go afterwards), and put it in the right group